File: Data Structures & Algorithms/missing-number/submission-2.js
class Solution {
	/**
	 * @param {number[]} nums
	 * @return {number}
	 */
	missingNumber(nums) {
		let [validSum, actuallSum] = [(nums.length * (nums.length + 1)) / 2, 0];
		for (const num of nums) {
			actuallSum += num;
		}
		return validSum - actuallSum;

	}
}

File: Data Structures & Algorithms/climbing-stairs/submission-2.py
class Solution:
    def climbStairs(self, n: int) -> int:
        if n==1 : return 1
        def dfs(num):
            if num > n:
                return 0
            if num == n:
                return 1
            return dfs(num+1) + dfs(num+2)
        return dfs(0)




